dashboard/test/ui/features/step_definitions/pd.rb

```diff
@@ -83,6 +93,46 @@
   user.permission = UserPermission::WORKSHOP_ORGANIZER
 end
 
+And(/^I make the teacher named "([^"]*)" a workshop admin$/) do |name|
+  require_rails_env
+
+  user = User.find_by(name: name)
+  user.permission = UserPermission::WORKSHOP_ADMIN
+end
+
+And(/^I create some fake applications of each type and state$/) do
```

Comment:
s/state/status. This is looping through `statuses`. We also have a concept of state (as in city, state) so this name is confusing.